.gitignore

```diff
@@ -40,7 +40,6 @@ server/manifests/*
 !server/manifests/README.md
 
 public/js/main*
-public/js/commonFramework*
 public/js/sandbox*
```

Comment:
What is this all about?  the `/public/js/*` files are GENERATED files, the source files are broken out into a bunch of different files under `/client/commonFramework`.

You need to make you changes to the appropriate sub file.